tests/test_platform_telegram.py

```diff
@@ -0,0 +1,35 @@
+import aiohttp
+import pytest
+
+from bottery.platform.telegram.api import TelegramAPI
+from bottery.platform.telegram.engine import TelegramEngine
+
+
+def test_platform_telegram_api_non_existent_method():
+    api = TelegramAPI('token', aiohttp.ClientSession)
+    with pytest.raises(AttributeError):
+        api.non_existent_method()
+
+
+@pytest.mark.asyncio
+async def test_platform_telegram_api_get_updates():
+    api = TelegramAPI('token', aiohttp.ClientSession)
+    with pytest.raises(TypeError):
+        await api.get_updates()
+
+
+def test_platform_telegram_engine_get_chat_id_private():
+    user = type('TelegramUser', (), {'id': '123'})
+    chat = type('TelegramChat', (), {'type': 'private'})
+    message = type('Message', (), {'user': user, 'chat': chat})
+    engine = TelegramEngine
+
+    assert engine.get_chat_id(engine, message) == '123'
+
+
+def test_platform_telegram_engine_get_chat_id_group():
+    chat = type('TelegramChat', (), {'id': '456', 'type': 'group'})
+    message = type('Message', (), {'chat': chat})
```

Comment:
Both test here are quite similar. I believe using a fixture to the Message instance and pytest parametrize feature would make it better. Something like (considering we have engine and message fixtures):
```
# this should work but I didn't test
@pytest.mark.parametrize('chat_type,id_expected', [
    ('group', 456),
    ('private', 123),
])
def test_platform_telegram_engine_get_chat_id(chat_type, id_expected, engine, message):
    setattr(message.chat, 'type', chat_type)
    assert engine.get_chat_id(message) == id_expected
```
What do you think?